frontend/src/lib/Quiver.test.ts

```diff
@@ -1014,17 +1014,17 @@ describe("Quiver", () => {
         const mockElement = { data: CATEGORICAL }
         const q = new Quiver(mockElement)
 
-        q.addRows(q)
+        const qq = q.addRows(q)
```

Comment:
This variable name seems a little weird to be honest.

Reply:
Counterpoint: this name feels appropriately descriptive to me (it's about adding "q" to "q") :)

Reply:
I agree with Tim here. `q1q2` is used elsewhere, when concatenating 2 different DataFrames.